flink-filesystems/flink-hadoop-fs/src/test/java/org/apache/flink/runtime/util/HadoopUtilsTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.util;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.security.UserGroupInformation;
+import org.apache.hadoop.security.UserGroupInformation.AuthenticationMethod;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Unit tests for Hadoop utils.
+ */
+public class HadoopUtilsTest {
+
+	@Test
+	public void testShouldNotCheckKerberosCredentialsForOtherAuthMethods() throws Exception {
+		UserGroupInformation.setConfiguration(getHadoopConfigWithAuthMethodOtherThanKerberos());
+		UserGroupInformation user = UserGroupInformation.createUserForTesting("test-user", new String[]{"user-group"});
+
+		boolean result = HadoopUtils.isCredentialsConfigured(user, true);
+
+		assertTrue(result);
+	}
+
+	private static Configuration getHadoopConfigWithAuthMethodOtherThanKerberos() {
```

Comment:
we can probably generalize this so we can test both side 
```
Configuration getHadoopConfigWithAutMethod(AuthenticationMethod method) {
  // ...
```

Reply:
done